tutorials/MD/README.md

```diff
@@ -1,31 +0,0 @@
-# Oxford Computational Biochemistry Molecular Dynamics Tutorial
```

Comment:
We're going to need a README.md for this tutorial set (even if it's something basic like what we have in the homology-modelling tutorial). I suggest we just add it post merge of this PR though. As long as the setup.md exists then at least students know what to do.